cloudsim-plus/src/main/java/org/cloudsimplus/faultinjection/VmClonerSimple.java

```diff
@@ -0,0 +1,82 @@
+package org.cloudsimplus.faultinjection;
+
+import org.cloudbus.cloudsim.cloudlets.Cloudlet;
+import org.cloudbus.cloudsim.vms.Vm;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.UnaryOperator;
+
+/**
+ * A basic implementation of a {@link VmCloner}.
+ *
+ * @author raysaoliveira
+ * @since CloudSim Plus 1.2.2
+ */
+public class VmClonerSimple implements VmCloner {
+    private UnaryOperator<Vm> vmClonerFunction;
+    private Function<Vm, List<Cloudlet>> cloudletsClonerFunction;
+    private int maxClonesNumber;
+    private int clonedVmsNumber;
+
+    /**
+     * Creates a {@link Vm} cloner which creates the maximum of 1 clone.
```

Comment:
Change "Creates a {@link Vm} cloner which creates the maximum of 1 clone." to "Creates a {@link Vm} cloner which makes the maximum of 1 Vm clone."